test/test_model_options.py

```diff
@@ -1,3 +1,4 @@
+import io
 import pytest
```

Comment:
please add a blank line here. See https://www.python.org/dev/peps/pep-0008/#imports for more informations ;)